doc/translatable.md

```diff
@@ -93,6 +93,12 @@ on how to setup and use the extensions in most optimized way.
 - **@Gedmo\Mapping\Annotation\Locale or @Gedmo\Mapping\Annotation\Language** this will identify this column as **locale** or **language**
 used to override the global locale
 
+### Translatable attributes:
+- **Gedmo\Mapping\Annotation\Translatable** it will **translate** this field
+- **Gedmo\Mapping\Annotation\TranslationEntity(class: MyClass::class)** it will use this class to store **translations** generated
```

Comment:
```suggestion
- **Gedmo\Mapping\Annotation\TranslationEntity(class: MyClass::class)** it will use this class to store generated **translations**
```